pages/commands/rush_add.md

```diff
@@ -17,16 +17,18 @@ be automatically prepended with a tilde, unless the "--exact" or "--caret"
 flags are used. The "--make-consistent" flag can be used to update all
 packages with the dependency.
 
-Optional arguments:
-  -h, --help            Show this help message and exit.
+Required arguments:
   -p PACKAGE, --package PACKAGE
-                        (Required) The name of the package which should be
```

Comment:
@chrispaterson thanks for submitting this PR. We overlooked it because the website has been moved to a new monorepo: 

But the file you are fixing is actually generated from the Rush CLI `--help`.  So we need to fix Rush first.  Here is a PR:  https://github.com/microsoft/rushstack/pull/3583